api/types/types.go

```diff
@@ -140,15 +155,18 @@ type ContainerPathStat struct {
 	Mtime time.Time   `json:"mtime"`
 }
 
+// ContainerProcessList contains response of Remote API:
 // GET "/containers/{name:.*}/top"
 type ContainerProcessList struct {
 	Processes [][]string
 	Titles    []string
 }
 
+// Version contains response of Remote API:
+// GET "/version"
 type Version struct {
 	Version       string
-	ApiVersion    version.Version
+	APIVersion    version.Version
```

Comment:
Same as above, as it is *serialized* as JSON, I think it would be better to do something like :

```go
APIVersion version.Version `json:ApiVersion`
```